app/js/Routes.js

```diff
@@ -101,6 +101,11 @@ export default (
 
       <Route path="/press" component={PressPage} />
 
+      <Route path="/devs" component={() => {
```

Comment:
we should do this at the server level instead of in javascript. will push a commit